### PR TITLE
Fixed an issue with a blank line in the VPN list in the indicator

### LINKED ISSUE #195: Odin 6: Wingpanel network shows strange vpn connection
Using Odin 6 updated to Nov 21st, Wingpanel network shows an empty vpn connection 
imagen

if you try to remove it , it doesn't exist in vpn panel

imagen

### PR DESCRIPTION
**Pull request by camellan**
Fixes #195

### CODE REVIEW

File: src/Widgets/VpnInterface.vala [hunk 1]
```diff
@@ -135,6 +136,9 @@ public class Network.VpnInterface : Network.WidgetNMInterface {
         if (active_vpn_item != null) {
             vpn_item.active = true;
         }
+        else{
```
Comment (jeremypw) [inline src/Widgets/VpnInterface.vala:139]: Elementary code style is
```
if () {

} else {

}
```

**Comment by jeremypw** [comment]
@camellan Thanks for the contribution!  Does this fix a reported issue (e.g. #195)? Is so please put "Fixes #nnn" in the description so that the issue is automatically closed when this is merged.

File: src/Widgets/VpnInterface.vala [hunk 2]
```diff
@@ -134,6 +135,8 @@ public class Network.VpnInterface : Network.WidgetNMInterface {
         check_vpn_availability ();
         if (active_vpn_item != null) {
             vpn_item.active = true;
+        } else {
+            vpn_item.active = false;
         }
```
Comment (itoshkov) [inline src/Widgets/VpnInterface.vala:140]: My knowledge of Vala is limited and my knowledge of the accepted coding practices in elementary OS -- even more so, so this suggestion might not be a good one.

But I'm experienced developer working mostly in Java lately, and I think the following is easier to read:

```suggestion
        vpn_item.active = active_vpn_item != null;
```

Comment (camellan) [reply src/Widgets/VpnInterface.vala:140]: Yes. It's much more elegant that way.

**Comment by camellan** [comment]
Ok. Thank you. I'll keep that in mind for the future.

**Comment by itoshkov** [comment]
I installed the code from @camellan's branch. It compiled without issues and it fixed the problem at least for me.

N.B. use `killall io.elementary.wingpanel` to restart wingpanel to test without having to logout and back in.

**Comment by jeremypw** [comment]
@camellan I cannot test this myself but on the basis of @itoshkov 's comment I would be willing to approve as it seems unlikely to causes any regressions.  The appdata needs updating for the next release before merging - if you do not know how to do that I can push a suitable commit later today.

**Comment by camellan** [comment]
https://github.com/camellan/wingpanel-indicator-network/blob/e1c5aef45445421f9bb851230b3a6698f8349a26/src/Widgets/VpnInterface.vala#L136

**Comment by camellan** [comment]
> if you do not know how to do that I can push a suitable commit later today.

Ok

**Review by jeremypw — APPROVED** [review]

**Pull request merged**